modules/translations/pages/index.adoc

```diff
@@ -16,6 +16,72 @@ The texts provided are not final and are subject to change. In particular,
 languages other than English have often been obtained via automated translation, and have not been reviewed yet. We will continue to add and update procurement labels in every future release of the SDK.
 ====
 
+[#index]
+== Index file
+
+`translations.json` is the translations folder index file, it contains two arrays documented below.
```

Comment:
We should not describe the "what", but rather explain the information provided. So something like:
"The file `translations.json` provides information on the translations files available in this folder, and the list of languages for which translations are provided."

Reply:
I put something close to what you suggested.